### ISSUE TITLE
Install options for non conda environments

### ISSUE THREAD

**Issue by aterhorst** [quanteda/spacyr#229]
I prefer Pycharm and Spyder to Conda. Using this command on my Mac:

```
spacy_install_virtualenv(python_path = "/Users/XXX/PycharmProjects/ner/venv/bin/python3.10",
                         python_version = "3.10",
                         lang_models = c("en_core_web_sm"))
```
results In this error message:

```
Error: Prerequisites for installing spaCy not available.

Execute the following at a terminal to install the prerequisites:
$ sudo /usr/bin/easy_install pip
$ sudo /usr/local/bin/pip install --upgrade virtualenv
```

I have pip working in my Pycharm virtual environment.  I guess ```spacy_install_virtualenv()``` is not recognising the path?

**Comment by amatsuo** [comment]
Although it's not our recommended solution, you can try installing it without spacyr. 

1. Activate virtualenv
2. `pip install spacy` 
3. `python -m spacy download en_core_web_sm`

Let us know the results!

**Comment by aterhorst** [comment]
I already had SpaCy installed. I addressed the problem by setting my environment variable this way:

```
Sys.setenv(PATH = paste("/Users/XXXPycharmProjects/ner/venv/bin", Sys.getenv("PATH"), sep = .Platform$path.sep),
           VIRTUAL_ENV = "/Users/XXX/PycharmProjects/ner/venv/")

spacy_initialize(model = "en_core_web_sm")

```
Everything works now

**Comment by bearT2** [comment]
Hello! I also had SpaCy installed in: 
"/Users/xxx/opt/anaconda3/lib/python3.8/site-packages"
What I should do to initialize spacy? I tried to revise your codes but was confused about what paths I should write within `paste()` and behine`VIRTUAL_ENV=`

I'm still new to Python and R. I've tried the spacy_install() and spacy_initialize(model = "en_core_web_sm") for hours, but I keep receiving mistakes...

Thank you! 

>
